source/_components/sensor.teleinfo.markdown

```diff
@@ -0,0 +1,103 @@
+---
+layout: page
+title: "EDF Teleinfo"
+description: "Instructions on how to integrate an EDF teleinfo sensor into Home Assistant."
+date: 2017-11-15 19:08
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: edf.png
+ha_category: Sensor
+ha_iot_class: "Energy"
+ha_release: "0.80"
+---
+
+The `teleinfo` sensor will read data stream from french electric meters. (Description in french
+[here](http://www.enedis.fr/sites/default/files/Enedis-NOI-CPT_02E.pdf).
+
+To enable this sensor, add the following lines to your `configuration.yaml` file:
+
+{% raw %}
+```yaml
+# Example configuration.yaml entry
+
+- platform: teleinfo
+  name: "edf"
```

Comment:
```suggestion
  name: "edf"
```
This is a optional variable, and in the first example config we want only required items.